src/core/MIGRATION.md

```diff
@@ -314,6 +315,43 @@ First, decouple your plugin's business logic from the dependencies that are not
 
 Once those things are finished for any given plugin, it can officially be switched to the new plugin system.
 
+### Shared application plugins
+
+Some services have been already moved to the new platform.
+
+Below you can find their new locations:
+
+| Service | Old place                                    | New place in the NP                                             |
+| --------------- | ----------------------------------------- | --------------------------------------------------- |
+| *FieldFormats*         | ui/registry/field_formats      | plugins/data/public |
+
+The `FieldFormats` service has been moved to the `data` plugin in the New Platform. If your plugin has any imports from `ui/registry/field_formats`, you'll need to update your imports as follows:
+
```

Comment:
Is there supposed to be a code that shows how to update imports?

Reply:
<img width="654" alt="Screen Shot 2019-11-20 at 12 28 16 PM" src="https://user-images.githubusercontent.com/16738889/69226493-4be9d300-0b91-11ea-9596-e352f0984b77.png">

Reply:
@restrry ^^